Makefile

```diff
@@ -31,8 +32,8 @@ LDFLAGS += -X $(PROJECT_PKG)/internal/version.GitCommit=${GIT_COMMIT}
 LDFLAGS += -X $(PROJECT_PKG)/internal/version.GitTreeState=${GIT_DIRTY}
 
 .PHONY: test
-test: vendor check-encoding
-	go test -race -v -coverprofile=coverage.txt -covermode=atomic ./...
+test: tidy vendor check-encoding
```

Comment:
There is a problem. If someone opens a PR and messes the `go.mod` and `go.sum` files, calling `go mod tidy` in `make test` may not detect this kind of issue in the PR check.

Reply:
I'm not entirely sure what you mean by messes. Changes?

Reply:
Yes, but it seems no harm for now.